erdpy/config.py

```diff
@@ -51,14 +51,54 @@ def get_dependency_url(key: str, tag: str, platform: str) -> str:
 
 def get_value(name: str) -> str:
     _guard_valid_name(name)
-    data = read_file()
+    data = get_active()
     return data.get(name, get_defaults()[name])
 
 
 def set_value(name: str, value: Any):
     _guard_valid_name(name)
     data = read_file()
-    data[name] = value
+    active_config = data.get("active", "default")
+    data.setdefault("configurations", {})
+    data["configurations"].setdefault(active_config, {})
+    data["configurations"][active_config][name] = value
+    write_file(data)
+
+
+def get_active():
+    data = read_file()
+    configs = data.get("configurations", {})
+    active_config = data.get("active", "default")
+
+    return configs.get(active_config, {})
+
+
+def set_active(name: str):
+    data = read_file()
+    _guard_valid_config_name(data, name)
+    data["active"] = name
+    write_file(data)
+
+
+def create_new(name: str, template: str):
```

Comment:
Suggest renaming function to `create_new_config()`.

Reply:
Done